packages/ui-kit/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,25 @@
+import classnames from 'classnames'
+import React from 'react'
+import { useCombinedRefsCallback } from '../../helpers'
+import { DefaultThemes, useTheme } from '../ThemeProvider'
+import componentStyles from './Card.module.scss'
+
+export interface CardProps extends React.ComponentPropsWithoutRef<'div'> {
+  baseTheme?: DefaultThemes
+}
+
+export const Card = React.forwardRef<HTMLDivElement, CardProps>(
+  ({ children, className, baseTheme, ...other }, forwardedRef) => {
+    const innerRef = React.useRef<HTMLDivElement>(null)
+    const { componentContainer, setRef } = useCombinedRefsCallback({ innerRef, forwardedRef })
```

Comment:
A way to have `inner` and `outer` refs for the component. The `inner` ref can be used for internal component logic and required for theming logic.